lib/tasks/import_csv.rake

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+# require './app/models/application_record'
+# require './app/models/customer'
+desc "import data from CSVs"
+namespace :import do
+  task :customers => :environment do
+    Customer.destroy_all
+    CSV.foreach('lib/customers.csv', headers: true) do |row|
+      row = row.to_hash
+      Customer.new(
+        :id => row['id'],
+        :first_name => row['first_name'],
+        :last_name => row['last_name'],
+        :created_at => row['created_at'],
+        :updated_at => row['updated_at'],
+      )
+
```

Comment:
I think I need to add a way to seed the data here, within each task. Going to create my resources for each of the other CSV files first then come back to this problem.

Reply:
@brianzanti

Reply:
Rather than using `new` here use `create`. Then I would have a print statement at the end of the task so you know if it has completed. Then do this for each of your files. You could also just pass the row to Customer without having to break it out unless there is information you are not importing from the CSV or if you've named an attribute differently than what the header was in the file. That doesn't look to be the case with Customer, but you may want to double-check your other files.